### DISCUSSION TITLE
I need a simple sip-proxy server

### DISCUSSION THREAD

**Discussion by nkozubsky** [resiprocate/resiprocate#236] [Q&A]
I need to connect some sip-phones to asterisk. I can't do it directly because there is no routes and firewall policy. 
If it possible with resiprocate as proxy, please show me a simple configuration. I not need web administration and TLS.
My schema:
SIP phone (10.1.1.0/24 UDP)<------->(10.1.1.1:5066 UDP) SIP Proxy (192.168.1.1/30 UDP) <--------> (192.168.1.2/30 UDP) Asterisk
The asterisk do not see 10.1.1.0/24 network at all.

Thanks.

**Comment by sgodin** [comment]
It is possible using repro routes.  I don't have time to create a configuration for you though.  

I would suggest reading the following:
https://github.com/resiprocate/resiprocate/wiki/docs/Repro_1.8_Overview.pdf
https://github.com/resiprocate/resiprocate/wiki/Using-Repro#Adding_Routes_to_Gateways
http://rtcquickstart.org/guide/multi/sip-proxy-repro.html#idp26

Scott
